src/openloco/ui/dropdown.cpp

```diff
@@ -149,6 +150,36 @@ namespace openloco::ui::dropdown
         call(0x004CC807, regs);
     }
 
+    /**
+     * 0x004CCDE7
+     *
+     * @param x
+     * @param y
+     * @param width
+     * @param height
+     * @param colour
+     * @param count
+     * @param columnCount
+     * @param heightOffset
+     */
+
+    void show_image(int16_t x, int16_t y, int16_t width, int16_t height, int16_t heightOffset, colour_t colour, uint8_t columnCount, uint8_t count)
+    {
+        assert(count < std::numeric_limits<uint8_t>::max());
```

Comment:
Count should probably also be below the size of _appropriateImageDropdownItemsPerRow?